invokeai/frontend/web/src/common/hooks/useClearStorage.ts

```diff
@@ -0,0 +1,12 @@
+import { idbKeyValStore } from 'app/store/store';
+import { clear } from 'idb-keyval';
+import { useCallback } from 'react';
+
+export const useClearStorage = () => {
+  const clearStorage = useCallback(() => {
+    clear(idbKeyValStore);
+    localStorage.clear();
```

Comment:
Can you still clear out local storage the same way in your browser tools?

Reply:
Local storage will still have some things and you'd do `localStorage.clear()` still. These are the keys still in local storage:
- `@@invokeai-color-mode` - chakra color mode, I can't figure out how to store this in redux, so it stores itself
- `ROARR_FILTER` - filter for the console logger
- `ROARR_LOG` - boolean to enable logging for console logger

For the new IndexedDB store, you can run `indexedDB.deleteDatabase('invoke')` but it won't work if the db is open - which is going to be never. 

Instead, to clear it manually, you go to browser's dev tools -> storage (FF) or application (chromium) ->